docs/sources/writing-guide/topic-types/tutorial/index.md

```diff
@@ -96,6 +96,49 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
 
 If getting access to the tutorial data is complex, include the instructions in the steps of the tutorial. If getting access to the data is straightforward, include it in the "Before you begin" section.
 
+## Where to locate your tutorial
+
+Sometimes it makes sense to publish your tutorial directly within the documentation for your product or feature. If you want higher visibility for your tutorial, you can add it to the main Grafana [Tutorials](/tutorials) page. Even though you're displaying your tutorial on the Tutorials page, you need to add your tutorial source to your team's repo to make it easy for team members to review the content. 
```

Comment:
nit: Splitting sentences onto new lines makes future line based diffing easier to read:
```suggestion
Sometimes it makes sense to publish your tutorial directly within the documentation for your product or feature.
If you want higher visibility for your tutorial, you can add it to the main Grafana [Tutorials](/tutorials) page. 
Even though you're displaying your tutorial on the Tutorials page, you need to add your tutorial source to your team's repo to make it easy for team members to review the content. 
```